codecov_auth/admin.py

```diff
@@ -310,6 +311,66 @@ class AccountAdmin(AdminMixin, admin.ModelAdmin):
         "is_delinquent",
     ]
 
+    @admin.action(description="Link Users to Account")
+    def link_users_to_account(self, request, queryset):
+        for account in queryset:
+            all_plan_activated_user_ids = set()
+            for org in account.organizations.all():
+                all_plan_activated_user_ids.update(set(org.plan_activated_users))
+
+            all_plan_activated_users = Owner.objects.filter(
+                ownerid__in=all_plan_activated_user_ids
+            ).prefetch_related("user")
+            non_student_count = all_plan_activated_users.exclude(student=True).count()
```

Comment:
Optional: maybe adding a comment on why we exclude students. Thanks for being so attentive and catching this this early, this is a edge case for suuuure

Reply:
hmmm, not really an edge case - check out `activated_user_count` or `can_activate_user` methods on `Owner` and `Account` - it's our built in policy.
I'm following the methods defined on the models to calculate things like `can_activate_user` - but having to write it out instead of using the pre-built methods so that I can do bulk operations 👍

Reply:
Right, what I meant was that this is an easy thing to slip since the majority of our customers aren't students, so I was praising you for being diligent, looking at previous code and learning about our policy